### PR TITLE
Fix integration python tests

### PR DESCRIPTION
**Pull request by aleksuss**
## Overview

---

See: https://jira.bf.local/browse/ECR-XYZW

### Definition of Done

- [ ] There are no TODOs left in the merged code
- [ ] Change is covered by automated tests
- [ ] Benchmark results are attached (if applicable)
- [ ] The [coding guidelines] are followed
- [ ] Public API has proper documentation
- [ ] Changelog is updated if needed (in case of notable or breaking changes)
- [ ] The continuous integration build passes

[coding guidelines]: https://github.com/exonum/exonum/blob/master/CONTRIBUTING.md#conventions

### CODE REVIEW

File: test-suite/exonum-py-tests/exonum_tests/crypto_advanced.py [hunk 1]
```diff
@@ -51,11 +63,14 @@ def test_create_wallet(self):
...
                     subscriber.wait_for_new_event()
-                self.assertEqual(crypto_client.get_wallet_info(alice_keys).status_code, 200)
+                self.assertEqual(
+                    crypto_client.get_wallet_info(alice_keys).status_code, 200
+                )
                 # TODO: Sometimes it fails without time.sleep() [ECR-3876]
                 time.sleep(2)
-                alice_balance = (crypto_client.get_wallet_info(alice_keys).json()
-                ['wallet_proof']['to_wallet']['entries'][0]['value']['balance'])
+                alice_balance = crypto_client.get_wallet_info(alice_keys).json()[
```
Comment (Kormick) [inline test-suite/exonum-py-tests/exonum_tests/crypto_advanced.py]: Strange formatting.

Comment (aleksuss) [reply test-suite/exonum-py-tests/exonum_tests/crypto_advanced.py]: Formatting by `black`. What's wrong with it ?

Comment (popzxc) [reply test-suite/exonum-py-tests/exonum_tests/crypto_advanced.py]: I agree that it looks very weird (seems that `black` isn't prepared for such a complex cases). Maybe split in several lines?

```python
wallet_info = crypto_client.get_wallet_info(alice_keys).json()
wallet_entry = wallet_info["wallet_proof"]["to_wallet"]["entries"][0]
alice_balance = wallet_entry["value"]["balance"]
```
?
```

File: test-suite/exonum-py-tests/exonum_tests/crypto_advanced.py [hunk 2]
```diff
@@ -75,8 +89,9 @@ def test_token_issue(self):
                     subscriber.wait_for_new_event()
                     # TODO: Sometimes it fails without time.sleep() [ECR-3876]
                     time.sleep(2)
-                    alice_balance = (crypto_client.get_wallet_info(alice_keys).json()
-                                     ['wallet_proof']['to_wallet']['entries'][0]['value']['balance'])
+                    alice_balance = crypto_client.get_wallet_info(alice_keys).json()[
+                        "wallet_proof"
```
Comment (popzxc) [inline test-suite/exonum-py-tests/exonum_tests/crypto_advanced.py]: Same here and below :)

**Pull request merged**